frontend/vite-env.d.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+interface ImportMetaEnv {
+  readonly VITE_BACKEND_URL: string;
+  readonly VITE_API_DOCS_URL: string;
+  readonly VITE_DOCS_URL: string;
+  readonly VITE_PROMETHEUS_URL: string;
+  readonly VITE_GRAFANA_URL: string;
+  readonly VITE_SMTP_MOCK_URL: string;
+  BASE_URL: string;
+  MODE: string;
+  DEV: boolean;
+  PROD: boolean;
+  SSR: boolean;
+}
+
```

Comment:
```suggestion

/// <reference types="vite/client" />

interface ImportMetaEnv {
  readonly VITE_BACKEND_URL: string;
  readonly VITE_API_DOCS_URL: string;
  readonly VITE_DOCS_URL: string;
  readonly VITE_PROMETHEUS_URL: string;
  readonly VITE_GRAFANA_URL: string;
  readonly VITE_SMTP_MOCK_URL: string;
}
```
We can use the default `vite/client` types which include the built-in environment variables and automatically update.